bitsong/chain.json

```diff
@@ -13,26 +13,26 @@
   "node_home": "$HOME/.bitsongd",
   "codebase": {
     "git_repo": "https://github.com/bitsongofficial/go-bitsong",
-    "recommended_version": "v0.15.0",
+    "recommended_version": "v0.20.4",
     "compatible_versions": [
-      "v0.15.0"
+      "v0.20.4"
     ],
     "binaries": {
-      "linux/amd64": "https://github.com/bitsongofficial/go-bitsong/releases/download/v0.15.0/bitsongd"
+      "linux/amd64": "https://github.com/bitsongofficial/go-bitsong/releases/download/v0.20.4/bitsongd"
```

Comment:
If including `binaries` in the chain.json, it must match the current version, which has two binaries defined (amd and arm), whereas there is only one (just amd) defined here.

Reply:
The binaries here still need to match